packages/emotion-server/package.json

```diff
@@ -20,8 +23,10 @@
     "emotion": "^9.1.3"
   },
   "devDependencies": {
+    "@types/react-dom": "16.0.5",
```

Comment:
It should depend on `@types/react-dom` as dependencies, or, at least as peerDependencies, shouldn't it?

Reply:
The only use is inside the tests so I don't think it should.

Reply:
@mgroenhoff Oh, you're right. My miss.